ESokha/08/Lesson_8_task_4_part_1.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>Lesson_8_task_4_part_1</title>
+        <script src="https://code.jquery.com/jquery-3.3.1.min.js">
+        </script>
+        <script>
+            $(document).ready(function()
+            {     
+                $('#passw, #passw2').on("keyup", function()
+                {
+                var passww = $("#passw").val();
+                var passww2 = $("#passw2").val();
+                    if(passww != passww2) {
+                        $("#show").text("Incorrect password").css('color', 'red');
+                        $("#passw").css('background-color', 'red');
```

Comment:
You can use #passw,#passw2 as selector here

Reply:
ok, I will correct